R/plot.R

```diff
@@ -557,7 +558,10 @@ plot.xts <- function(x,
     }
   }
   assign(".xts_chob", cs, .plotxtsEnv)
-  cs
+  plot(cs)
```

Comment:
This forces the plot to be rendered. The user may not want that, because it can be time consuming to re-render the plot if another panel is added.